### PR TITLE
FAB-1253 connections and experiments example

### PR DESCRIPTION
**Pull request by ljha-CS**
Note: We can merge this only after https://github.com/CognitiveScale/cortex-python/pull/51 is merged

### CODE REVIEW

File: RestApi/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Job Webhook example (cortex/Job_Webhook_Example)
```
Comment (jgielstra-cs) [inline RestApi/README.md:1]: Needs some rework why is it mentioned webhook example

File: RestApi/agent.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+camel: 1.0.0
+name: cortex/Web_Hook_Example
```
Comment (jgielstra-cs) [inline RestApi/agent.yaml:2]: Webhook ?

File: connection/Makefile [hunk 3]
```diff
@@ -0,0 +1,33 @@
+IMAGE_TAG=latest
```
Comment (jgielstra-cs) [inline connection/Makefile:1]: Are we consistently using Makefile or shell script ?

File: connection/main.py [hunk 4]
```diff
@@ -0,0 +1,33 @@
+from fastapi import FastAPI
+from logzero import logger
+from pymongo import MongoClient
```
Comment (jgielstra-cs) [inline connection/main.py:3]: Perhaps showing this at init/startup for a daemon would be better that at invoke ?

File: experiment-predict/README.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
+### Daemon Skill type
+
+Long running Cortex skill serving REST API requests
+
```
Comment (jgielstra-cs) [inline experiment-predict/README.md:4]: Make README.md relevant to example ?

File: experiment-predict/main.py [hunk 6]
```diff
@@ -0,0 +1,19 @@
+from cortex import Cortex
+from fastapi import FastAPI
+
+app = FastAPI()
+
```
Comment (jgielstra-cs) [inline experiment-predict/main.py:5]: Perhaps show at startup/init versus invoke ??

File: experiment-train/README.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
+### Cortex Experiment Train SKill
+
+Cortex Skill to train save model in Experiment
```
Comment (jgielstra-cs) [inline experiment-train/README.md:3]: More words /description of what is going on why use this ... what it does ?

**Review by jgielstra-cs — APPROVED** [review]
We can merge still needs some work..
Thing we should use makefile consistently versus that and deploy/sh
Readme's should be more descriptive per example

File: connection/Makefile (hunk 3 above)
Comment (ljha-CS) [reply connection/Makefile:1]: Used Makefile to run on *nix and windows (GNUMake). Skill generator also updated to use Makefile https://github.com/CognitiveScale/cortex-cli/pull/304

File: connection/main.py (hunk 4 above)
Comment (ljha-CS) [reply connection/main.py:3]: Is this approach good, or should I create new `/init` endpoint and demonstrate usage using an agent? Issue with `/init` endpoint is, when we scale daemons only one pod gets initialized and others fail to serve request

File: experiment-train/README.md (hunk 7 above)
Comment (lkrishna-cs) [reply experiment-train/README.md]: Agreed.  Need a paragraph on why would one use this Skill. Also, any pre-reqs to be documented.

File: experiment-predict/main.py (hunk 6 above)
Comment (lkrishna-cs) [reply experiment-predict/main.py:5]: Definitely need to add init() to load the model. Loading it in predict() should not be a best practice.

**Review by lkrishna-cs — COMMENTED** [review]
Added some comments for experiments usage in predict.

**Comment by pkandarpa-cs** [comment]
@ljha-CS if you've addressed all the review comments made by @lkrishna-cs and @jgielstra-cs , we should merge this PR

File: experiment-train/skill.yaml [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+      - name: params
+        type: object
+        description: Input Params
+        required: true
+    routing:
+      all:
+        action  : experiment-train
+        runtime: cortex/jobs
+        output: response
+
```
Comment (ssingh-CS) [inline experiment-train/skill.yaml:18]: output name of action and skill should be same else the skill will not get saved.

**Review by ssingh-CS — APPROVED** [review]

**Pull request merged**